src/librados/IoCtxImpl.h

```diff
@@ -158,7 +158,7 @@ struct librados::IoCtxImpl {
   int operate_read(const object_t& oid, ::ObjectOperation *o, bufferlist *pbl, int flags=0);
   int aio_operate(const object_t& oid, ::ObjectOperation *o,
 		  AioCompletionImpl *c, const SnapContext& snap_context,
-		  int flags, const blkin_trace_info *trace_info = nullptr);
+		  int flags, const jspan_context *trace_info = nullptr);
```

Comment:
please keep blkin for now

Reply:
assume you mean we can add the jspan_context, but leave blkin for later removal?  I don't want us to have to do an unsafe cast to/from a blkin pointer type

Reply:
removing blkin separately somehow feels cleaner, but at the same time, I've confirmed with core that blkin can be unceremoniously removed, just saying

Reply:
> assume you mean we can add the jspan_context, but leave blkin for later removal? I don't want us to have to do an unsafe cast to/from a blkin pointer type

agree. lets add another function (for now)